yhondanilo/chess.rb

```diff
@@ -0,0 +1,46 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila, @columna=fila, columna
+    intercalar
+  end
+  def intercalar
+    @board=Array.new(@fila){Array.new(@columna)}
+    @board[0][0]=1
+    for f in 0..self.fila-1
```

Comment:
Prefer `each` over `for`.<br>Redundant `self` detected.<br>Surrounding space missing for operator '-'.